content/en/docs/collector/transforming-telemetry.md

```diff
@@ -0,0 +1,128 @@
+
+The attributes processor can be used to update, insert, delete, or replace existing attributes on metrics or traces. For example, here’s a configuration that adds an attribute called account_id to all spans:
+
+```
+processors:
+  attributes/accountid:
+    actions:
+      - key: account_id
+        value: 2245
+        action: insert
+```
+
+The resource processor has an identical configuration, but applies only to [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md). Use the resource processor to modify infrastructure metadata related to telemetry. For example, this inserts the Kubernetes cluster name:
+
+```
+processors:
+  resource/k8s:
+    attributes:
+    - key: k8s.cluster.name
+      from_attribute: k8s-cluster
+      action: insert
+```
+
+More advanced attribute transformations are also available in the [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor).
+
+## Renaming Metrics or Metric Labels
+
+**Processor:** [metrics transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor)
+
+The [metrics transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor) shares some functionality with the [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor), but also supports renaming and other metric-specific functionality.
+
+```
+processors:
+  metricstransform/rename:
+    transforms:
+      include: system.cpu.usage`
+      action: update
+      new_name: system.cpu.usage_time
+```
+
+The [metrics transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor) also supports regexes to apply transform rules to multiple metric names or metric labels at the same time. This example renames cluster_name to cluster-name for all metrics:
+
+
+```
+processors:
+  metricstransform/clustername:
+   transforms:
+     - include: "^.*$"
+       match_type: regexp
+       action: update
+       operations:
+       - action: update_label
+         label: cluster_name
+         new_label: cluster-name
+```
+
+## Enriching Telemetry with Resource Attributes
+
+**Processor**: [resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor) and [k8sattributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor)
+
+It is recommended that end-users use processors to enrich their telemetry data with relevant infrastructure metadata to help teams quickly identify when underlying infrastructure is impacting service health or performance.
```

Comment:
I'd remove the recommendation part and consider for example:
"It is recommended that end-users use processors to enrich their telemetry data with..."
->
"These processors can be used for enriching telemetry with..."
(this is to keep the style similar to other parts in this document)